src/components/icon/icon.tsx

```diff
@@ -34,8 +34,19 @@ import { IconSize } from './icon.types';
  * ```
  * <limel-config config={{iconPath: '/my/parent/path/'}} />
  * ```
- *
- * ## Icon Finder
+ * ### Lime icons
+ * There are icons included in **@limetech/lime-icons8** package which are
```

Comment:
```suggestion
 * There are icons included in **@lundalogik/lime-icons8** package which are
```

Because that package is private, since we don't have a license to redistribute the icons from Icons8.

I've already fixed this locally, but wanted to explain why.

Reply:
tnx